src/components/Footer/Footer.stories.mdx

```diff
@@ -28,25 +28,128 @@ import DSProvider from "../../theme/provider";
 
 | Component Version | DS Version |
 | ----------------- | ---------- |
-| Added             | `1.1.0`    |
-| Latest            | `1.1.0`    |
+| Added             | `TBD`      |
+| Latest            | `TBD`      |
 
 ## Table of Contents
 
 - [Overview](#overview)
 - [Component Props](#component-props)
 - [Accessibility](#accessibility)
+- [With DS Template Components](#with-ds-templates)
 
 ## Overview
 
 <Description of={Footer} />
 
+This component is expected to be rendered at the end of a webpage, before the
+closing `</body>` tag. There are no expected props for this component so it can
+be rendered simply as `<Footer />`.
+
+```tsx
+<body>
+  <header>...</header>
+  <main>...</main>
+
+  <!-- Add the Footer component here -->
+  <Footer />
+</body>
+```
+
 ## Component Props
 
 <Canvas withToolbar>
-  <Story name="Footer">{(args) => <Footer />}</Story>
+  <Story name="Footer">
+    <Footer />
+  </Story>
 </Canvas>
 
-<ArgsTable story="Footer" />
+Note: There are custom props for the `Footer` component.
 
 ## Accessibility
+
+The `Footer` component renders as an HTML `<footer>` element with a WAI-ARIA role
+of `role="contentinfo"`. This is the common practice for the footer landmark
+region on a page to maximize its accessibility in browsers and assistive
+technologies.
+
+Rendering this `Footer` component, or similar HTML landmark region components,
+inside another landmark region causes accessibility issues. For example,
+rendering the `Footer` component inside an `aside` HTML element is an error.
+
+Resources:
+
+- [W3C WAI Page Structure Code Example](https://www.w3.org/WAI/tutorials/page-structure/example/)
+- [W3C WAI Page Regions](https://www.w3.org/WAI/tutorials/page-structure/regions/)
+
+## With DS Template Components
+
+The `Footer` component is a page layout component and it can be used within the
+DS `Template`s components.
+
+### TemplateAppContainer Component
+
+The DS' `TemplateAppContainer` component renders an HTML `<footer>` element when
+content is passed to its `footer` prop. If your application uses the
+`TemplateAppContainer` component and the `Footer` component, make sure to also
+set the `renderFooterElement` prop to `false`. Both components will render HTML
+`<footer>` elements and we only need one; setting `renderFooterElement` to
+`false` will prevent rendering nested `<footer>` elements.
+
+If this step is not done, the `TemplateAppContainer` component will log a warning
```

Comment:
As you did with the `Individual Template Components` example, move this text below the example code block.